tasks/section_01_level1.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: ensure copy banner's
+  copy:
+    src: "etc/{{ item }}"
+    dest: "/etc/{{ item }}"
+    mode: 0644
+    owner: root
+  with_items:
+    - issue
+    - issue.net
+    - motd
+  tags: section01
+
+- name: ensure remove insecure kernel modules
+  modprobe:
+    name: "{{ item }}"
+    state: absent
+  with_items:
+    - cramfs
+    - freevxfs
+    - jffs2
+    - hfs
+    - hfsplus
+    - udf
+  tags: section01
+
+- name: ensure blacklist insecure kernel modules
+  copy:
+    src: "etc/{{ item }}"
+    dest: "/etc/{{ item }}"
+    mode: 0644
+    owner: root
+  with_items:
+    - modprobe.d/cis-section_01_level1.conf
```

Comment:
I don't understand the reason to use a cis-section as file name here, to copy to VM to this `/etc/...` path. It's not can cause any issue inside a VM?

Reply:
This just is a reference for we remember why disable this modules =)

Reply:
nice!